docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -169,4 +445,111 @@ TBD
 Examples
 ~~~~~~~~
 
-TBD
+The examples that follow use JSON Schema file ``schema.jsonschema`` with content
+
+.. code:: sh
+  {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+      "values": {
+        "type": "object",
+        "properties": {
+          "greeting": {
+            "type": "string"
+          },
+          "recipient": {
+            "type": "string"
+          }
+        },
+        "required": ["greeting", "recipient"],
+        "additionalProperties": false
+      }
+    },
+    "required": ["values"],
+    "additionalProperties": false
+  }
+
+and YAML file ``values.yaml`` with content
+
+.. code:: sh
+
+  values:
+    greeting: Hello
+    recipient: World
+
+* Validate a YAML config against a given JSON schema:
+
+  .. code:: sh
+
+    $ uw config validate --schema-file schema.jsonschema --input-file values.yaml
+    [2024-01-03T17:23:07]     INFO 0 schema-validation errors found
+
+  Shell redirection via ``|``, ``>``, et al may also be used to stream output to a file, another process, etc.
+
+
+* Read the config from ``stdin`` and render to ``stdout``:
+
+  .. code:: sh
+
+    $ cat values.yaml | uw config validate --schema-file schema.jsonschema
+    [2024-01-03T17:26:29]     INFO 0 schema-validation errors found
+
+
+* However, you cannot read the schema from ``stdin`` and render to ``stdout``:
+
+  .. code:: sh
+
+    $ cat schema.jsonschema | uw config validate -input-file values.yaml
+    uw config validate: error: the following arguments are required: --schema-file
+
+* If there are differences between the config files, they will be shown with the schema. For example, with ``recipient: World`` removed from ``values.yaml``:
```

Comment:
Maybe this needs an update after copy/paste? This isn't about "differences between the config files", it's about a config failing validation.